web/sharings/sharings.go

```diff
@@ -11,6 +11,40 @@ import (
 	"github.com/labstack/echo"
 )
 
+// SharingAnswer handles a sharing answer from the sharer side
+func SharingAnswer(c echo.Context) error {
+
+	var err error
+	var sharingAccepted bool
+
+	state := c.FormValue("state")
+	clientID := c.FormValue("client_id")
+	scope := c.FormValue("scope")
+	accessCode := c.FormValue("access_code")
+
+	instance := middlewares.GetInstance(c)
+
+	// The sharing is refused if there is no access code or scope
+	if scope == "" || accessCode == "" {
+		sharingAccepted = false
+	} else {
+		sharingAccepted = true
+	}
```

Comment:
Yes, I used this notation first but I wasn't sure the shortness wouldn't harm the readability, but maybe it is just because I'm not quite used to it